rosdep/python.yaml

```
@@ -8424,6 +8424,16 @@ python3-pyvista-pip:
   ubuntu:
     pip:
       packages: [pyvista]
+python3-pyxb-x-pip:
+  debian:
+    pip:
+      packages: [PyXB-X]
+  osx:
+    pip:
+      packages: [PyXB-X]
+  ubuntu:
+    pip:
+      packages: [PyXB-X]
```

I believe this is a good place to use the wildcard.

```suggestion
python3-pyxb-x-pip:
  '*':
    pip:
      packages: [PyXB-X]
```